thoth/common/logging.py

```diff
@@ -214,6 +237,18 @@ def init_logging(
         for logger in ignored_loggers.split(","):
             ignore_logger(logger)
 
+    ignored_exceptions = os.getenv("THOTH_SENTRY_IGNORE_EXCEPTION")
+    if ignored_exceptions:
+        exceptions_split = ignored_exceptions.split(',')
+        for exception in exceptions_split:
+            exception_parts = exception.rsplit('.', maxsplit=1)
+            if len(exception_parts) == 2:
+                exc_module, exc_name = exception_parts
+                _IGNORED_EXCEPTIONS.append((exc_module, exc_name))
+            else:
+                root_logger.error(
+                    "Configuration for exceptions to be ignored not set correctly.")
```

Comment:
Generally, it's good to report what's wrong/what code could not parse. It helps with debugging later on. :)

Reply:
Let me add that here, one min!

Reply:
You are challenged with bots :robot: They merge on their own once the PR is accepted. :)